aptos-move/framework/aptos-framework/sources/block.spec.move

```diff
@@ -71,6 +73,14 @@ spec aptos_framework::block {
         let addr = signer::address_of(aptos_framework);
         let account = global<account::Account>(addr);
         aborts_if account.guid_creation_num + 2 >= account::MAX_GUID_CREATION_NUM;
+
+        // During the module's initialization, it guarantees that the BlockResource resource move under the Aptos framework account with initial values.
```

Comment:
Please double check whether the spec here is overlapped with `spec schema Initialize`.

Reply:
There is one property overlapped